akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -34,11 +80,16 @@ object ORMap {
  */
 @SerialVersionUID(1L)
 final class ORMap[A, B <: ReplicatedData] private[akka] (
-  private[akka] val keys:   ORSet[A],
-  private[akka] val values: Map[A, B])
-  extends ReplicatedData with ReplicatedDataSerialization with RemovedNodePruning {
+  private[akka] val keys:          ORSet[A],
+  private[akka] val values:        Map[A, B],
+  private[akka] val tombstoneVals: Map[A, B],
```

Comment:
@gosubpl I looked through this again and the part I don't fully understand is the tombstones. When are they removed (well, I see that they are removed at several places)?